metadata-ingestion/src/datahub/ingestion/source/schema/json_schema.py

```diff
@@ -78,6 +80,14 @@ class JsonSchemaSourceConfig(StatefulIngestionConfigBase, DatasetSourceConfigMix
         default=None,
         description="Use this if URI-s need to be modified during reference resolution. Simple string match - replace capabilities are supported.",
     )
+    inject_default_description: bool = Field(
+        default=True,
+        description="When enabled, injects a default description if one is not present in the schema",
+    )
+    inject_empty_description: bool = Field(
```

Comment:
does description being `None` vs "" really change anything? I don't really get the motivation for this flag